app/helpers/d3-callback.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export function d3Callback([callback, key]/*, hash*/) {
```

Comment:
maybe this function isn't necessary... we can do it with a helper we've already defined?